package.json

```diff
@@ -2,19 +2,20 @@
   "name": "itowns",
   "version": "0.0.0-alpha",
   "description": "A JS/WebGL framework for 3D geospatial data visualization",
-  "main": "dist/itowns.js",
+  "main": "lib/Main.js",
   "scripts": {
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc src/Core/Commander/Interfaces/ApiInterface/*",
     "doclint": "npm run doc -- -t templates/silent",
     "test": "npm run lint && npm run build && npm run doclint",
     "build": "webpack -p",
     "start": "webpack-dev-server -d --inline --hot",
-    "prepublish": "npm run build"
+    "prepublish": "npm run lint && npm run doc && npm run build && BABEL_DISABLE_CACHE=1 babel src --out-dir lib"
```

Comment:
Where does this (`npm run doc`) generate the documentation? shouldn't it rather be `npm run doclint` instead (only to ensure the jsdoc is OK)?  
Do we package the doc in the NPM module? (and if so, should we? I'm not quite sure…)